### ISSUE TITLE
[Feature Request] Option to make ZNC/*status NOTICE instead of PRIVMSG [$5]

### ISSUE THREAD

**Issue by incognico** [znc/znc#328]
It is a matter of taste but I think having *status NOTICE is less annoying, for example in this case:

06:43 <_status> IRC connection timed out.  Reconnecting...
06:43 <_status> Disconnected from IRC. Reconnecting...
06:44 <_status> IRC connection timed out.  Reconnecting...
06:46 <_status> IRC connection timed out.  Reconnecting...
06:48 <_status> IRC connection timed out.  Reconnecting...
06:50 <_status> IRC connection timed out.  Reconnecting...
06:51 <_status> Connected!
06:53 <_status> The channel #xy could not be joined, disabling it.

Sometimes NOTICE is used already, in this case parting #xy:
11:43 -*status(znc@znc.in)- Removing channel [#xy]

My wish: Let the user choose if he wants to get PRIVMSG'd or NOTICE'd
##

**Comment by incognico** [comment]
@teward that depends on the client and its configuration, I don't think this is a valid concern, especially not if PRIVMSG/NOTICE is toggleable as an option in ZNC and you default to PRIVMSG (even if it would be wrong to do (at least in the case of the lines I have posted in op), see @Mikaela's post).

**Comment by jpnurmi** [comment]
:+1:

I'd also prefer such messages as "Connected!" as notices, instead of opening a new *status query window.

**Comment by dreamcat4** [comment]
+1

I'd also prefer such messages as "Connected!" as notices, instead of opening a new *status query window.

**Comment by dreamcat4** [comment]
Hi. In the meantime I have a suggested a workaround for weechat client, which is actually a bug fix:

https://github.com/weechat/scripts/issues/146

Unfortunately it seems they are being highly resistive to the idea of merging it ATM. Heres to hoping they will be a bit more sympathetic, and possibly change mind / make a concession. Its just a couple of lines of code.

**Comment by unixtam** [comment]
> My wish: Let the user choose if he wants to get PRIVMSG'd or NOTICE'd

I know @incognico's original request is almost six years old, but I ran into the same issue and decided to scratch my itch. I have just submitted PR #1645, which adds a new module hook, and introduces a new user module called "statusnotice". Users can activate this module in order to make *status send notices instead of privmsgs.

**Comment by nyuszika7h** [comment]
I agree with this, the `*status` constant reconnection, and `*perform` (because I have a trigger in my client to call perform when a specified nick comes online) can be annoying. It should simply be an option, that avoids the concern of "it may not be shown properly by all clients". People who enable the option will take care of properly configuring their client.
